components/internal-services/kustomization.yaml

```diff
@@ -0,0 +1,9 @@
+resources:
+- internal_service_request_role.yaml
+- internal_service_request_role_binding.yaml
+- internal_service_request_service_account.yaml
```

Comment:
`components/internal-services/internal_service_service_account_token.yaml` should be included as well

Reply:
added